docs/source/diagrams/IqtFitMultiple-v1_wkflw.dot

```diff
@@ -71,4 +71,7 @@ digraph IqtFitMultiple {
   SampleLogs                    -> AddSampleLogMultiple         [label="Additional SampleLogs based on Fit data"]
   AddSampleLogMultiple          -> ResultWorkspace2             [label="Outputworkspace"]
   AddSampleLogMultiple2         -> FitGroup2                    [label="OutputWorkspace"]
+  ExtractMembers                -> ExtractQENSMembers           [label="Extract members when true"]
```

Comment:
Very good to update the diagrams.

A suggestion first: would it be easier and quicker to read "Extract members ?"

In `IqtFitSequential`, the colors are different (yellow and blue), here they are gray. I think it should be the same for both algorithms. I guess `IqtFitSequential` has the desired look of this part of the diagram.